### ISSUE TITLE
Bootstrap is throwing exception on Edge in Image feature guide

### ISSUE THREAD

**Issue by mlewand** [ckeditor/ckeditor5#1974]
## Is this a bug report or feature request?

📃 Docs

## 💻 Version of CKEditor

CKEditor v5 @ latest `master` (12.4.0 release candidate)

## 📋 Steps to reproduce

There are at least two different exception thrown from Bootstrap in our [Image feature guide](https://ckeditor.com/docs/ckeditor5/latest/features/image.html#responsive-images) when using Edge.

The case from gif is the following:

1. Open sample
2. Scroll to the "Resizing images" section
3. Select last image
4. Copy it to clipboard
5. Put the selection at the beginning of the document
6. Insert a 3x2 table
7. Paste the widget into 1x2 and 2x1 cell.
8. Try to resize last image.

## ✅ Expected result

No exceptions thrown.

## ❎ Actual result

There are two different errors:

```
0: Unable to get property 'contains' of undefined or null reference
```

and

```
0: Unable to get property '0' of undefined or null reference
```

**Comment by mlewand** [comment]
Actually it's highly possible that it was #1957 - but Edge shown that it originate in bootstrap.js.

**Comment by mlewand** [comment]
The issue also happens on Chrome and there the issue is:

```
contextualballoon.js:311 Uncaught TypeError: Cannot read property '0' of undefined
    at m._getStackId (contextualballoon.js:311)
    at m.remove (contextualballoon.js:253)
    at l._hideToolbar (widgettoolbarrepository.js:169)
    at l._updateToolbarsVisibility (widgettoolbarrepository.js:143)
    at f.<anonymous> (widgettoolbarrepository.js:77)
    at fire (emittermixin.js:207)
    at f.update (editorui.js:94)
    at g.<anonymous> (editorui.js:65)
    at g.fire (emittermixin.js:207)
    at Tt.<anonymous> (view.js:190)
```

I'm now building the docs for the latest editor to check if it has been fixed.

**Comment by mlewand** [comment]
Confirmed that the issue has been fixed, so it's a duplciate of #1974.
